src/docs/ftw-analytics/how-to-set-up-analytics-for-ftw/index.md

```diff
@@ -1,61 +1,132 @@
-> environment with `yarn run dev-server`.
+Google recently released their new analytics service Google Analytics 4.
+Support for Google Universal Analytics will end on October 1, 2023. New
+versions of FTW provide out-of-the-box support for Google Analytics 4.
 
-## Optional: Use a custom analytics handler
+If you're starting development on a new version of FTW and prefer to use
+Universal Analytics, you should look into how Analytics was implemented
+before
+[this pull request](https://github.com/sharetribe/ftw-daily/pull/1508).
+If you have built your marketplace on top of an older version of FTW and
+you want to start using Google Analytics 4, you'll need to implement the
+changes introduced in
+[this pull request](https://github.com/sharetribe/ftw-daily/pull/1508).
 
-If you want to add a new analytics library, you can do as follows:
+> New versions of FTW will expect a Tracking ID compatible with Google
+> Analytics 4. FTW expects the ID to begin with the "G-" prefix.
 
-### 1. Add the analytics library script
+#### Enhanced measurements
 
-If the analytics library has an external script, add the library script
-tag to the
+It is not recommended to use the Enhanced Measurements feature
+introduced in Google Analytics 4, which is enabled by default. The
+Enhanced Measurements feature injects code into link tags which can
+break in-app navigation in FTW. Therefore, we strongly recommend
+disabling the Enhanced Measurements feature when using Google Analytics
+4 with FTW.
+
+<video>
+    <source src='./turn-off-enhanced-measurements.mp4' type='video/mp4'>
+    <source src='./turn-off-enhanced-measurements.webm' type='video/webm'>
+    <source src='./turn-off-enhanced-measurements.ogv' type='video/ogg'>
+</video>
+
+If that's not an option, you can continue to use Enhanced Measurements
+if you disable the _Outbound clicks_ and _page changes based on browser
+history events_ features.
+
+![Disable Outbound clicks](./disable.png)
+
+### Built-in handlers
+
+FTW includes an
+[event handler](https://github.com/sharetribe/ftw-daily/blob/89b9390e7235253067d0e78d9f838fbd6b07c10d/src/analytics/handlers.js#L16)
+that sends `page_view` events to Google Analytics. These events need to
+registered manually because FTW is a single-page application, meaning
+that in-app navigation does not render a page load.
+
+The Google Analytics script registers a `page_view` event automatically
+on every page load. The
+[`trackPageView`](https://github.com/sharetribe/ftw-daily/blob/89b9390e7235253067d0e78d9f838fbd6b07c10d/src/analytics/handlers.js#L16)
+function takes this into account and only sends a `page_view` event to
+Google if a page is accessed through in-app navigation.
+
+If you'd like to track something other than page views, you can
+implement your custom handler using the `trackPageView` function as an
+example.
+
+## Custom analytics libraries
+
+If you choose to go with another analytics provider, you can follow
```

Comment:
```suggestion
If you choose to add another analytics provider (e.g. Facebook Pixel), you can follow
```

It might be worth it to mention Facebook Pixel since it's asked quite often.